lms/djangoapps/cohort_management_dashboard/static/cohort_management_dashboard/templates/student_table_body.underscore

```diff
@@ -0,0 +1,20 @@
+  <% for ( var i = 0; i < users.length; i++ ) { %>
+      <tr class="row row_number_<%- i%2 %>">
+        <td><%- users[i].username %></td>
+          <td>
+
+            <select class="action-cohort-select" name="select">
+                <% for ( var j = 0; j < cohorts.length; j++ ) { %>
+                    <option
+                        value="<%- cohorts[j].id %>" <%- (cohorts[j].id == currentCohortID)? 'selected':'' %> >
```

Comment:
I would really appreciate if we can add some spaces between the ternary operators, thank you in advance.

Reply:
fixed